tests/base.py

```diff
@@ -227,7 +224,7 @@ def modify_expected_datatypes(self, expected_records):
 
     def align_date_type(self, record, key, value):
         """datetime values must conform to ISO-8601 or they will be rejected by the gate"""
-        if isinstance(value, str) and key in ['updated_at']:
+        if isinstance(value, str) and key in ['updated_at', 'created_at']:# TODO: update to reflect replication keys
```

Comment:
Should this TODO be done in this PR?

Reply:
no, @kspeer825 asked me to put it there for him